<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopManagerForm</class>
 <widget class="QWidget" name="ShopManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
<<<<<<< HEAD
=======
     <item row="1" column="1">
      <widget class="QTreeWidget" name="treeWidget">
       <property name="rootIsDecorated">
        <bool>false</bool>
<<<<<<< HEAD
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>CID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Count</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Price</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Total Price</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Address</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Phone Number</string>
        </property>
       </column>
      </widget>
     </item>
>>>>>>> 06db11e7a6a4eec6acd82a0723527bc089e7bb97
     <item row="0" column="1">
      <widget class="QTableView" name="shopTableView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
<<<<<<< HEAD
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
=======
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
=======
<<<<<<< HEAD
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>CID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Count</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Price</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Total Price</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Address</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Phone Number</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTableView" name="shopTableView"/>
=======
       </property>
<<<<<<< HEAD
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>CID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Count</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Price</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Total Price</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Address</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Phone Number</string>
        </property>
       </column>
=======
      </column>
      <column>
       <property name="text">
        <string>CID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Count</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Phone Number</string>
       </property>
      </column>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="inputPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>276</width>
         <height>314</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Input</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="shopFormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="idLabel">
            <property name="text">
             <string>SID</string>
            </property>
            <property name="buddy">
             <cstring>shopsidLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="shopsidLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="shopcidcomboBox"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="categoryLabel">
            <property name="text">
             <string>Date</string>
            </property>
            <property name="buddy">
             <cstring>addressLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="dateLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Count</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="countLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="priceLabel">
            <property name="text">
             <string>PName</string>
            </property>
            <property name="buddy">
             <cstring>phoneNumberLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="shopLabel">
            <property name="text">
             <string>Name</string>
            </property>
            <property name="buddy">
             <cstring>nameLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="shoppidcomboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Client</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="CustomertreeWidget">
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Phone Number</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Address</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Product</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="ProducttreeWidget">
          <column>
           <property name="text">
            <string>Product Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Price</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Category</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="addHorizontalLayout">
          <item>
           <widget class="QPushButton" name="addPushButton">
            <property name="text">
             <string>&amp;Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="modifyPushButton">
            <property name="text">
             <string>&amp;Modify</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
>>>>>>> d909dbfecd081b130e3216b38300aa7a71178669
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTableView" name="tableView"/>
>>>>>>> 5c7d7596cc6e7c0746ae023420c18bb27f3356ad
>>>>>>> d4bb97a7110e1e254903e178b9e5539cff2110d3
     </item>
>>>>>>> 06db11e7a6a4eec6acd82a0723527bc089e7bb97
    </layout>
   </item>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
<<<<<<< HEAD
      <number>1</number>
=======
      <number>0</number>
>>>>>>> 06db11e7a6a4eec6acd82a0723527bc089e7bb97
     </property>
     <widget class="QWidget" name="inputPage">
      <property name="geometry">
       <rect>
        <x>0</x>
<<<<<<< HEAD
        <y>0</y>
=======
<<<<<<< HEAD
        <y>-53</y>
=======
        <y>0</y>
>>>>>>> d4bb97a7110e1e254903e178b9e5539cff2110d3
>>>>>>> 06db11e7a6a4eec6acd82a0723527bc089e7bb97
        <width>269</width>
        <height>314</height>
       </rect>
      </property>
      <attribute name="label">
       <string>&amp;Input</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="shopFormLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="idLabel">
           <property name="text">
            <string>SID</string>
           </property>
           <property name="buddy">
            <cstring>shopsidLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="shopsidLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="shopcidcomboBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="categoryLabel">
           <property name="text">
            <string>Date</string>
           </property>
           <property name="buddy">
            <cstring>addressLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="dateLineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Count</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="countLineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="priceLabel">
           <property name="text">
            <string>PName</string>
           </property>
           <property name="buddy">
            <cstring>phoneNumberLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="shopLabel">
           <property name="text">
            <string>Name</string>
           </property>
           <property name="buddy">
            <cstring>nameLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="shoppidcomboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Client</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="CustomertreeWidget">
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Phone Number</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Address</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Product</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="ProducttreeWidget">
         <column>
          <property name="text">
           <string>Product Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Category</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="addHorizontalLayout">
         <item>
          <widget class="QPushButton" name="addPushButton">
           <property name="text">
            <string>&amp;Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="modifyPushButton">
           <property name="text">
            <string>&amp;Modify</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="searchPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>279</width>
        <height>261</height>
       </rect>
      </property>
      <attribute name="label">
       <string>&amp;Search</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
<<<<<<< HEAD
        <widget class="QTableView" name="searchTableView">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
=======
<<<<<<< HEAD
        <widget class="QTableView" name="searchTableView"/>
       </item>
       <item>
=======
>>>>>>> d4bb97a7110e1e254903e178b9e5539cff2110d3
        <widget class="QTreeWidget" name="searchTreeWidget">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Produce Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Count</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Total Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Phone Number</string>
          </property>
         </column>
>>>>>>> 06db11e7a6a4eec6acd82a0723527bc089e7bb97
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="searchComboBox">
           <item>
            <property name="text">
<<<<<<< HEAD
             <string>SID</string>
=======
             <string>ID</string>
>>>>>>> 06db11e7a6a4eec6acd82a0723527bc089e7bb97
            </property>
           </item>
           <item>
            <property name="text">
<<<<<<< HEAD
             <string>CID</string>
=======
             <string>Name</string>
>>>>>>> 06db11e7a6a4eec6acd82a0723527bc089e7bb97
            </property>
           </item>
           <item>
            <property name="text">
<<<<<<< HEAD
             <string>PID</string>
=======
             <string>Product Name</string>
>>>>>>> 06db11e7a6a4eec6acd82a0723527bc089e7bb97
            </property>
           </item>
           <item>
            <property name="text">
             <string>Date</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Count</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="searchLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="searchPushButton">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
